MAUI/ListView/emptyview.md

```diff
@@ -0,0 +1,320 @@
+
+ * EmptyView - object can be set to a string or view when `SfListView` has no items. The default value is null.
+ * EmptyViewTemplate - used to customize the appearance of `EmptyView`. The default value is null.
+
+## Display a string when listview has no items
+
+The `SfListView.EmptyView` property can be set to a string, which will be displayed when the `ItemsSource` is null, or when the collection specified by the `ItemsSource` property is null or empty.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<ContentPage xmlns:syncfusion="clr-namespace:Syncfusion.Maui.ListView;assembly=Syncfusion.Maui.ListView">
+  <syncfusion:SfListView x:Name="listView"
+                         ItemsSource="{Binding Items}"
+                         ItemSize="56"
+                         EmptyView="No Items">
+  </syncfusion:SfListView>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+listView.EmptyView = "No Items";
+{% endhighlight %}
+{% endtabs %}
+
+## Display views when listview has no items
+
+The `SfListView.EmptyView` property can be set to a view, which will be displayed when the `ItemsSource` property is null, or when the collection specified by the ItemsSource property is null or empty.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<ContentPage xmlns:syncfusion="clr-namespace:Syncfusion.Maui.ListView;assembly=Syncfusion.Maui.ListView">
+  <syncfusion:SfListView x:Name="listView"
+                         ItemsSource="{Binding Items}"
+                         ItemSize="56">
+    <syncfusion:SfListView.EmptyView>
+      <StackLayout VerticalOptions="CenterAndExpand" >
+        <Label Text="&#xe725;" FontSize="40" HorizontalTextAlignment="Center"
+                               FontFamily="{OnPlatform iOS=ListViewFontIcons, MacCatalyst=ListViewFontIcons, Android=ListViewFontIcons.ttf#, UWP=ListViewFontIcons.ttf#ListViewFontIcons}" />                      
+        <Label Text="No Items" FontSize="16" FontFamily="Roboto-Regular" HorizontalTextAlignment="Center" />
+      </StackLayout>                    
+    </syncfusion:SfListView.EmptyView>                       
+  </syncfusion:SfListView>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+StackLayout stackLayout = new StackLayout() { VerticalOptions = LayoutOptions.CenterAndExpand };
+
+var label1 = new Label()
+{
+  Text = "\ue725",
+  FontSize = 40,
+  HorizontalTextAlignment = TextAlignment.Center,
+  FontFamily = "ListViewFontIcons.ttf#"
+};
+var label2 = new Label()
+{
+  Text = "No Items",
+  FontSize = 16,
+  FontFamily = "Roboto-Regular",
+  HorizontalTextAlignment = TextAlignment.Center,
+};
+stackLayout.Children.Add(label1);
+stackLayout.Children.Add(label2);
+
+listView.EmptyView = stackLayout;
+{% endhighlight %}
+{% endtabs %}
+
+N> View displayed by `SfListView.EmptyView` can be a single view or a view that contains multiple child views.

```

Comment:
N> View displayed by the `SfListView.EmptyView` can be a single view or a view that contains multiple child views.